contrib/http_servlet/src/main/java/io/opencensus/contrib/http/servlet/OcHttpServletFilter.java

```diff
@@ -145,15 +145,11 @@ public void init(FilterConfig filterConfig) throws ServletException {
       extractor = new OcHttpServletExtractor();
     }
 
-    obj = context.getAttribute(OC_PUBLIC_ENDPOINT);
+    String publicEndVal = context.getInitParameter(OC_PUBLIC_ENDPOINT);
```

Comment:
Does this also mean that getting Propagator and Extractor should also be changed?

Reply:
Maybe but I am not sure why an application developer would want to change that. Those are more purely within the realm of settings for tool configuration. I have not done testing related to those, like I have with publicEndVal.